includes/model/ShiftSignupState.php

```diff
@@ -82,18 +37,18 @@ public function combineWith(ShiftSignupState $shiftSignupState)
     }
 
     /**
-     * @param string $state
+     * @param ShiftSignupStatus $state
      * @return int
      */
-    private function valueForState($state)
+    private function valueForState(ShiftSignupStatus $state)
     {
         return match ($state) {
-            ShiftSignupState::NOT_ARRIVED, ShiftSignupState::NOT_YET, ShiftSignupState::SHIFT_ENDED => 100,
-            ShiftSignupState::SIGNED_UP => 90,
-            ShiftSignupState::FREE      => 80,
-            ShiftSignupState::ANGELTYPE, ShiftSignupState::COLLIDES => 70,
-            ShiftSignupState::OCCUPIED, ShiftSignupState::ADMIN     => 60,
-            default => 0,
+            ShiftSignupStatus::NOT_ARRIVED, ShiftSignupStatus::NOT_YET, ShiftSignupStatus::SHIFT_ENDED => 100,
+            ShiftSignupStatus::SIGNED_UP => 90,
+            ShiftSignupStatus::FREE      => 80,
+            ShiftSignupStatus::ANGELTYPE, ShiftSignupStatus::COLLIDES => 70,
+            ShiftSignupStatus::OCCUPIED,  ShiftSignupStatus::ADMIN    => 60,
+            default => 0, // TODO: shouldn't happen
```

Comment:
TODO or not TODO… :thinking: :wink:

Reply:
should i remove the line? the question is why was it there before?

Reply:
Having 0 here would imho eventually be usefull when extending the enum so keeping it (and removing the ToDo) would be enough.